include/forms/textBoxes.php

```diff
@@ -0,0 +1,19 @@
+<?php
+function singleLineTextBox($textTitle,$textName,$textPlaceholder, $value, $valueArray){
+    return $textTitle.": ".returnRequireStatement($valueArray)."<br/><input type='text' name='$textName' id='$textName' placeholder='$textPlaceholder' value = '" . rememberedValue($value)."' /><br/><br/>";
```

Comment:
I didn't talk about this when we went over forms, but there's one potential problem here. What if the value being passed to this form has a single quote in it? It'd look something like this:
`<input  type='text' value='it's a value' />`

See the problem? The single quote in `it's` would cause the `value` attribute to complete and you'd miss the `s a value`.

Check out the `htmlspecialchars` function for this. The same applies for the other form functions you have here.